.github/workflows/bitcoin-tests.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           - tests::neon_integrations::bitcoind_forking_test
           - tests::neon_integrations::should_fix_2771
           - tests::neon_integrations::pox_integration_test
-          - tests::neon_integrations::mining_events_integration_test
+          - tests::neon_integrations::mining_events_integration_test_FLAKY_TEST_FIX_ME
```

Comment:
This is *not* flaky.  It's broken in `develop` due to a recent change in how we report skipped transactions to the event observer.

Reply:
I'd be happy to approve this with this just marked `mining_events_integrations_test_BROKEN`

Reply:
It's about to be fixed in https://github.com/stacks-network/stacks-blockchain/pull/3204